pkg/odo/genericclioptions/add_flags.go

```diff
@@ -9,9 +9,20 @@ func AddOutputFlag(cmd *cobra.Command) {
 
 // AddContextFlag adds `context` flag to given cobra command
 func AddContextFlag(cmd *cobra.Command, setValueTo *string) {
+	helpMessage := "Use given context directory as a source for component settings"
 	if setValueTo != nil {
-		cmd.Flags().StringVar(setValueTo, "context", "", "Use given context directory as a source for component settings")
+		cmd.Flags().StringVar(setValueTo, ContextFlagName, "", helpMessage)
 	} else {
-		cmd.Flags().String("context", "", "Use given context directory as a source for component settings")
+		cmd.Flags().String(ContextFlagName, "", helpMessage)
+	}
+}
+
+// AddNowFlag adds `now` flag to given cobra command
+func AddNowFlag(cmd *cobra.Command, setValueTo *bool) {
+	helpMessage := "Push changes to cluster right now"
```

Comment:
Maybe: `Push changes to the cluster immediately` sounds better.

Reply:
Sure, updating